yellowbrick/target/feature_correlation.py

```diff
@@ -0,0 +1,87 @@
+# yellowbrick.classifier.feature_correlation
+# Class balance visualizer for showing per-class support.
+#
+# Author    Zijie (ZJ) Poh <poh.zijie@gmail.com>
+# Author:   Benjamin Bengfort <bbengfort@districtdatalabs.com>
+# Created:  Wed Jul 29 15:30:40 2018 -0700
+#
+# ID: feature_correlation.py [5388065] neal@nhumphrey.com $
+
+"""
+Feature Correlation to Dependent Variable Visualizer.
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import numpy as np
+
+from .base import TargetVisualizer
+
+from sklearn.feature_selection import (mutual_info_classif,
+                                       mutual_info_regression)
+from scipy.stats import pearsonr
+
+
+##########################################################################
+## Class Feature Correlation
+##########################################################################
+
+class FeatureCorrelation(TargetVisualizer):
+
+    method_func = {
```

Comment:
For static variables like this, I would prefer if they were all caps, e.g. `METHOD_FUNC` and `METHOD_LABEL`. Did you see an example implementation in YB for something like this where the names were lower case? If so, I can review why we did things that way, or fix it if it needs fixing!

Reply:
Yeap. There are three in `yellowbrick.features.rankd`.
<https://github.com/DistrictDataLabs/yellowbrick/blob/933bc47df0ce5b29792023d0023134d31fab30b9/yellowbrick/features/rankd.py#L191>
<https://github.com/DistrictDataLabs/yellowbrick/blob/933bc47df0ce5b29792023d0023134d31fab30b9/yellowbrick/features/rankd.py#L334>
<https://github.com/DistrictDataLabs/yellowbrick/blob/933bc47df0ce5b29792023d0023134d31fab30b9/yellowbrick/features/rankd.py#L448>

Reply:
Gah, yeh I suspected something like this existed. Bummer - I think the reason this is this way in RankD is because of the subclassing. 

Go ahead and keep it lower case for now and I'll figure out what to do with this later.  

Also, could we name this `correlation_method` and `correlation_label`?